layouts/_default/baseof.html

```diff
@@ -2,8 +2,14 @@
 <html lang="en">
     {{- partial "head.html" . -}}
     <body>
-        {{- partial "header.html" . -}}
-        <div id="content">
+        {{ $title := .Title }}
+        {{ if eq $title "Home"}}
+          {{- partial "header_home.html" . -}}
+        {{ end }}
+        {{ if ne $title "Home"}}
+          {{- partial "header.html" . -}}
+        {{ end}}
```

Comment:
You can do an [if..else](https://gohugo.io/templates/introduction/#example-4-if--else) in hugo for this whole part

Reply:
addressed in upcoming code push